facts/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from django.contrib.auth.models import User
+from topics.models import Topic
+
+
+class Fact(models.Model):
+    content = models.CharField(max_length=300)
+    topic = models.ForeignKey(Topic)
+    owner = models.ForeignKey(User)
+
+    @property
+    def score(self):
+        score = 0
+        for reaction in self.reaction_set.all():
+            score += 1 if reaction.is_upvote else -1
+        return score
+
+
+class Reaction(models.Model):
```

Comment:
So yes I was wondering about the duplication here, but it's doing a slightly different thing; these are reactions to facts, while what you did was reactions to articles. We can have some kind of abstract thing perhaps?